### PR TITLE
example for applymodel

### LINKED ISSUE #131: update the examples in MVG after latest changes in AMI

### PR DESCRIPTION
**Pull request by tuix**
# Description

Add an example to demonstrate how inference works by applying an existing model on new data.

New dependencies: (Added to requirements_*.txt)
`sklearn` added to requirements_doc.txt

closes #132  
closes #131

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue -> bump patch)
- [ ] New feature (non-breaking change which adds functionality -> bump minor version)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected -> bump major version)
- [x] Documentation Update

## Checklist

- [ ] I have added tests that prove that my fix/feature works
- [ ] Linters pass locally and I have followed PEP8 code style
- [ ] New and existing tests pass locally
- [ ] I have updated the documentation if needed
- [ ] I have commented hard-to-understand areas in the code

## Requirements

- [ ] I have updated the MVG version

### CODE REVIEW

File: docs/source/content/examples/8-inference.ipynb [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+    "session.wait_for_analyses(request_id_list=[request_id])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mode_output = session.get_analysis_results(request_id=request_data[\"request_id\"])"
```
Comment (vnadhan) [inline docs/source/content/examples/8-inference.ipynb:194]:
```suggestion
    "mode_output = session.get_analysis_results(request_id=request_id])"
```

**Review by vnadhan — CHANGES_REQUESTED** [review]
Looks good. Just a minor suggestion.

**Comment by vnadhan** [comment]
A comment - the example is not added in the Sphnix docs tree.

**Review by vnadhan — APPROVED** [review]

**Review by SergioMDCB — APPROVED** [review]
This PR incorporates the Inferance example into the documentation

**Pull request merged**